util/cinc-install-1.1.0.sh

```diff
@@ -259,6 +259,26 @@ do_download() {
   unable_to_retrieve_package
 }
 
+# get_region INSTANCE_METADATA_FILE
+# get region from metadata
+get_region() {
+  if exists curl; then
+    echo "Trying curl to get region with IMDSv2 ..."
+    token=$(curl -s --connect-timeout 5 --max-time 5 --retry 3 -X PUT "http://169.254.169.254/latest/api/token" -H "X-aws-ec2-metadata-token-ttl-seconds: 300")
+    region=$(curl -s -H "X-aws-ec2-metadata-token: ${token}" http://169.254.169.254/latest/dynamic/instance-identity/document|grep region|awk -F\" '{print $4}')
+  elif exists python; then
+    echo "Trying python to get region with IMDSv2 ..."
+    token=$(python -c "import sys,requests; sys.stdout.write(requests.put('http://169.254.169.254/latest/api/token', headers={'X-aws-ec2-metadata-token-ttl-seconds': '300', 'User-Agent': 'mixlib-install/3.11.27'}).content)")
+    region=$(python -c "import sys,requests,json; sys.stdout.write(json.loads(requests.get('http://169.254.169.254/latest/dynamic/instance-identity/document', headers={ 'X-aws-ec2-metadata-token': '${token}', 'User-Agent': 'mixlib-install/3.11.27' }).content.decode())['region'])")
+  fi
+
+  if test "x$region" = "x"; then
```

Comment:
nit: what about `if [ -z "$region" ]`, to be consistent with other bash conditionals we do in the repo. Same comment for other places where we are doing this `test "x$var" = "x"` logic

Reply:
In `cinc-install.sh`, all bash  condition are in `test "x$var" = "x"`, so I make it the same format here to be consistence